diff --git a/qt/aqt/__init__.py b/qt/aqt/__init__.py
index 53bdc3c92..747e90951 100644
--- a/qt/aqt/__init__.py
+++ b/qt/aqt/__init__.py
@@ -667,7 +667,7 @@ def _run(argv: list[str] | None = None, exec: bool = True) -> AnkiApp | None:
 
     # create the app
     QCoreApplication.setApplicationName("Anki")
-    QGuiApplication.setDesktopFileName("anki")
+    QGuiApplication.setDesktopFileName("net.ankiweb.Anki")
     app = AnkiApp(argv)
     if app.secondInstance():
         # we've signaled the primary instance, so we should close