diff --git a/qt/aqt/forms/main.ui b/qt/aqt/forms/main.ui
index 5b2af199c..3decde05f 100644
--- a/qt/aqt/forms/main.ui
+++ b/qt/aqt/forms/main.ui
@@ -93,7 +93,6 @@
     <addaction name="actionAdd_ons"/>
     <addaction name="separator"/>
     <addaction name="actionNoteTypes"/>
-    <addaction name="action_check_for_updates"/>
     <addaction name="actionPreferences"/>
    </widget>
    <widget class="QMenu" name="menuqt_accel_view">
@@ -284,11 +283,6 @@
     <string>qt_accel_load_backup</string>
    </property>
   </action>
-  <action name="action_check_for_updates">
-   <property name="text">
-    <string>addons_check_for_updates</string>
-   </property>
-  </action>
  </widget>
  <resources>
   <include location="icons.qrc"/>
diff --git a/qt/aqt/main.py b/qt/aqt/main.py
index c6998dc65..b1131a200 100644
--- a/qt/aqt/main.py
+++ b/qt/aqt/main.py
@@ -1339,6 +1339,8 @@ title="{}" {}>{}</button>""".format(
         aqt.dialogs.open("Preferences", self)
 
     def on_check_for_updates(self) -> None:
+        return
+
         from packaging.version import Version
 
         from aqt.update import get_latest_release_op, prompt_and_install_github_update
@@ -1469,7 +1471,6 @@ title="{}" {}>{}</button>""".format(
         qconnect(m.actionCreateFiltered.triggered, self.onCram)
         qconnect(m.actionEmptyCards.triggered, self.onEmptyCards)
         qconnect(m.actionNoteTypes.triggered, self.onNoteTypes)
-        qconnect(m.action_check_for_updates.triggered, self.on_check_for_updates)
         qconnect(m.actionPreferences.triggered, self.onPrefs)
 
         # View
diff --git a/qt/aqt/update.py b/qt/aqt/update.py
index 9121abaf6..7459cfb92 100644
--- a/qt/aqt/update.py
+++ b/qt/aqt/update.py
@@ -18,6 +18,8 @@ from aqt.utils import openLink, show_warning, showText, tr
 
 
 def check_for_update() -> None:
+    return
+
     from aqt import mw
 
     def do_check(_col: Collection) -> CheckForUpdateResponse:
