diff --git a/build/configure/src/python.rs b/build/configure/src/python.rs
index e43bceeb3..d2675fdb3 100644
--- a/build/configure/src/python.rs
+++ b/build/configure/src/python.rs
@@ -81,9 +81,7 @@ pub struct GenPythonProto {
 impl BuildAction for GenPythonProto {
     fn command(&self) -> &str {
         "$protoc $
-        --plugin=protoc-gen-mypy=$protoc-gen-mypy $
         --python_out=$builddir/pylib $
-        --mypy_out=$builddir/pylib $
         -Iproto $in"
     }
 
@@ -101,7 +99,6 @@ impl BuildAction for GenPythonProto {
             .collect();
         build.add_inputs("in", &self.proto_files);
         build.add_inputs("protoc", inputs![":protoc_binary"]);
-        build.add_inputs("protoc-gen-mypy", inputs![":pyenv:protoc-gen-mypy"]);
         build.add_outputs("", python_outputs);
     }
 
@@ -119,7 +116,7 @@ pub struct BuildWheel {
 
 impl BuildAction for BuildWheel {
     fn command(&self) -> &str {
-        "$uv build --wheel --out-dir=$out_dir --project=$project_dir"
+        "$uv build --wheel --out-dir=$out_dir --project=$project_dir --build-constraint /run/build/anki/uv-constraints.txt"
     }
 
     fn files(&mut self, build: &mut impl FilesHandle) {
 