diff --git a/qt/aqt/mpv.py b/qt/aqt/mpv.py
index b389df434..c70d845b4 100644
--- a/qt/aqt/mpv.py
+++ b/qt/aqt/mpv.py
@@ -93,9 +93,6 @@ class MPVBase:
 
     if is_win:
         default_argv += ["--af-add=lavfi=[apad=pad_dur=0.150]"]
-    if not is_mac:
-        # our mpv build for macOS doesn't support this option (compiled out)
-        default_argv += ["--no-ytdl"]
 
     def __init__(self, window_id=None, debug=False):
         self.window_id = window_id